### ISSUE TITLE
Feature to remove station from the starred a stops a list missing

### ISSUE THREAD

**Issue by petebp** [grote/Liberario#74]
It is only possible to remove recently used connection, but not to remove stops, which recently was used for departure-informations. Due that, the list will grow more and more with stops, which are used very frequently and makes the list a bit cluttered after some time of use.

**Comment by dktzde** [comment]
I would be really happy to see this enhancement in Liberario.

**Comment by grote** [comment]
For me personally this feature does not have a high priority since the favorite locations are sorted by how often you used them and you can also search them by typing in the form field.

However, if somebody wants to implement it this would as always be appreciated. Please let's just discuss the UI concept of this feature here before starting the implementation.

**Comment by sebalis** [comment]
Two points from #343 which was rightly closed as a duplicate: 1. ideally I would like to be able to both clear the entire list and remove indivdual entries (perhaps by swiping them to the right) and 2. this is also a privacy issue if people can watch me make a query (perhaps friends to whom I might want to show off this beautiful app).

**Comment by sebalis** [comment]
Having to run a few tests while inspecting ticket #421, I’d just like to mention again how positively infuriating it is to me that any location I enter (even those I never use to start a connection search) shows up in the history. It makes me recoil in horror. Throw the app out of the window. And more. No, I do not want to clear the whole app data or switch providers, I only want to remove those test entries. If I were able to start working on Transportr as a developer, I would give this ticket by far the highest priority. See #343 for details of how I envisage this. Sorry for the clutter but I had to get this off my chest.

**Comment by ialokim** [comment]
As this is still a missing functionality in version 2 of the app, I would like to ask for ideas on how to implement such a feature in the existing UI. Favorite locations are currently only showed in a dropdown menu while searching. @sebalis Do you have some ideas on how to realize the clearing functionality in this scenario?

**Comment by sebalis** [comment]
No, sorry, no suggestions from he. I did write earlier that this issue makes me want to throw app out of the window. Effectively, that is what I’ve done.

**Comment by Altonss** [comment]
The easiest would maybe just have an entry in the left panel like "Favorites" under the "Parameters". If you click on it simply opens a list of favorites that you can manage...
